addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/UserControlledOpenRedirectScanner.java

```diff
@@ -79,6 +83,15 @@ private void checkUserControllableLocationHeaderValue(
             return;
         }
 
+        String requestDomain = null;
+        try {
+            requestDomain = msg.getRequestHeader().getURI().getAuthority();
+        } catch (URIException ex) {
+            logger.error(
+                    "unable to get URI from Request. Ignoring and moving ahead with the scanning OpenRedirect",
+                    ex);
```

Comment:
Please change this to use a leading capital `Unable`. Also is it worth continuing here? If getAuthority() fails and requestDomain is null shouldn't we just return?

Reply:
As there are some cases where _decode method fails so just ignoring the exception and moving ahead.